### ISSUE TITLE
Loss goes to infinity on running with default setup

### ISSUE THREAD

**Issue by anantzoid** [edgarriba/ali-pytorch#1]
I just cloned the repo and ran the `main.py` with the parameters, as listed in README and got the following training progress:
```
[0/100][0/733] Loss_D: 1.38627135754 Loss_G: 1.38978719711
[0/100][1/733] Loss_D: 1.38632798195 Loss_G: 1.38739454746
[0/100][2/733] Loss_D: 1.38354945183 Loss_G: 1.38604617119
[0/100][3/733] Loss_D: 1.38877046108 Loss_G: 1.38658261299
[0/100][4/733] Loss_D: 1.42622494698 Loss_G: 1.35285282135
[0/100][5/733] Loss_D: 3.73983073235 Loss_G: 2.93968224525
[0/100][6/733] Loss_D: inf Loss_G: 19.790802002
[0/100][7/733] Loss_D: inf Loss_G: inf
[0/100][8/733] Loss_D: nan Loss_G: nan
```

**Comment by wanglouis49** [comment]
I tried to clip the weights and it will run a few epochs, but it seems this code simply does not work.

**Comment by edgarriba** [comment]
Hi, I got similar issues and I couldn't find the way to replicate the paper experiments and consequently I abandoned this project. If you find where is the issue, please let me know.
